examples/iota_lens/analysis_iotalens.py

```diff
@@ -96,20 +96,20 @@ def read_all_files(file_pattern):
     rsuffix='_initial'
 )
 # add new columns: dH and dI
-beam_joined['dH'] = beam_joined["H_initial"] - beam_joined["H_final"]
-beam_joined['dI'] = beam_joined["I_initial"] - beam_joined["I_final"]
+beam_joined['dH'] = abs(beam_joined["H_initial"] - beam_joined["H_final"])
+beam_joined['dI'] = abs(beam_joined["I_initial"] - beam_joined["I_final"])
```

Comment:
If we use the `.abs()` method of the resulting dataframe, then pandas might be able to compute this in delayed fashion (when we access it later - aka lazy computing).
```suggestion
beam_joined['dH'] = (beam_joined["H_initial"] - beam_joined["H_final"]).abs()
beam_joined['dI'] = (beam_joined["I_initial"] - beam_joined["I_final"]).abs()
```